pages/linux/cmus.md

```diff
@@ -11,6 +12,14 @@
 
```

Comment:
Your addition itself is good, but as we're here, I suggest to change `file/directory` above to `file or directory`.